examples/scale-out-cluster-local/config-cluster-member0.json

```diff
@@ -0,0 +1,31 @@
+{
+  "distSpecVersion": "1.1.0",
+  "storage": {
+    "rootDirectory": "./workspace/zot/data/mem1",
```

Comment:
We have to figure out a scheme to append a member path.
Would like to have a single zot configuration that folks don't have to tweak.

Reply:
Are you able to run a `zb` benchmark to show things scaling up?